README.md

```diff
@@ -378,6 +378,32 @@ UserModel:
     roles: List<string>
     customProperties: Map<String, Property>?
 ```
+since 6.4.0 inline types are supported now even when adding extra configuration:
```

Comment:
Update version & the rest of this readme as well. So the readme matches the latest version